GVFS/GVFS.Service/Handlers/EnableAndAttachProjFSHandler.cs

```diff
@@ -87,9 +87,26 @@ public static bool TryEnablePrjFlt(ITracer tracer, out string error)
                 isNativeLibInstalled = ProjFSFilter.IsNativeLibInstalled(tracer, new PhysicalFileSystem());
                 if (!isNativeLibInstalled)
                 {
-                    string missingNativeLibMessage = "Native library is not installed";
-                    error = string.IsNullOrEmpty(error) ? missingNativeLibMessage : $"{error}. {missingNativeLibMessage}";
-                    tracer.RelatedError($"{nameof(TryEnablePrjFlt)}: {missingNativeLibMessage}");
+                    if (isRunning)
+                    {
+                        tracer.RelatedInfo($"{nameof(TryEnablePrjFlt)}: Native ProjFS library is not installed, attempting to copy packaged version");

```

Comment:
Would it be interesting to see this in telemetry?  I don't believe Info logged there by default.

Reply:
I would say this should be a warning, since it's about an undesirable state that would cause problems, but we might be able to mitigate (copying the library file).

Reply:
suggestion: `attempting to copy packaged version` => `attempting to copy version packaged with VFS for Git`

'packaged version` can be ambiguous? but this is just for debugging purposes, so maybe it will be clear to us...

Reply:
> Would it be interesting to see this in telemetry? I don't believe Info logged there by default.

\+ @jrbriggs for his thoughts on this.  I can certainly update this code to send telemetry, but I'm not sure how we'd react to that data and I think we're trying to be frugal with the telemetry we're sending.

> I would say this should be a warning

I'll update this to a warning.

Reply:
I doubt this particular point would generate _too much_ data when compared with the rest of the telemetry exhaust, so I don't worry about the data volume perspective.

The reason to dispatch this is so we could understand how many (would-have-been) mount failures this logic is mitigating. However, I don't think the event has any value beyond that.

Reply:
@jrbriggs thanks! I will update this to be sent via telemetry.